PreMailer.Net/PreMailer.Net.Tests/PreserveEntitiesTests.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using Xunit;
+
+namespace PreMailer.Net.Tests
+{
+    public class PreserveEntitiesTests
+    {
+        [Fact]
+        public void MoveCssInline_GivenCopyEntity_PreservesEntity()
+        {
+            string htmlEncoded = "&copy;";
+            string input = $"<html><head></head><body><div>{htmlEncoded}</div></body></html>";
+            
+            var premailedOutput = PreMailer.MoveCssInline(input, preserveEntities: true);
+
+            Assert.Contains(htmlEncoded, premailedOutput.Html);
+            Assert.DoesNotContain("©", premailedOutput.Html);
+        }
+    }
+}
```

Comment:
Do not put tests into new classes to convey a use-case. Put tests where they locally make sense, in terms of how the code base is structured. This test case belongs to the premailer tests class.